docs/general/istiod-ha.md

```diff
@@ -0,0 +1,95 @@
+# Running Istiod in HA mode
+Please follow this guide to run Istiod in HA mode. By default, istiod is deployed with replica count set to 1, to be able to run it in HA mode, you can achieve it in two different ways:
+* Setting `replicaCount` to 2 or more in Istio resource and disabling autoscale (by default enabled).
+* Setting `autoscaleMin` to 2 or more in Istio resource and keeping `autoscaleMax` to 2 or more.
+
+Pros and Cons of each approach:
+- **Setting `replicaCount` to 2 or more**:
+  - Pros: Simplicity, easy to understand and manage.
+  - Cons: Fixed number of replicas, no autoscaling based on load.
```

Comment:
Since this method disables the pdb, maybe we should only recommend this for single node clusters or clusters with only one worker node?